CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
Fixed, see http://code.qt.io/cgit/qt/qtbase.git/tree/dist/changes-5.4.2 for details

I was requiring cmake 2.8.12 as rviz-qt5 and moveit-qt5 both apply against jade-devel (but the same fix should work there as well)